core/iwasm/common/wasm_c_api.c

```diff
@@ -1023,7 +1027,8 @@ wasm_externtype_delete(wasm_externtype_t *extern_type)
             wasm_tabletype_delete(wasm_externtype_as_tabletype(extern_type));
             break;
         default:
-            LOG_WARNING("%s meets unsupported type", __FUNCTION__, extern_type);
+            LOG_WARNING("%s meets unsupported type %u", __FUNCTION__,
+                        extern_type);
```

Comment:
Should be `wasm_externtype_kind(extern_type)`, here extern_type is a pointer of type `wasm_externtype_t *`.